webapp/src/modules/ui/preview/reducer.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { IPreviewController } from '@dcl/schemas'
+import { setWearablePreviewController, setEmotePlaying } from './actions'
+import { previewReducer, INITIAL_STATE, PreviewState } from './reducer'
+
+let state: PreviewState
+
+beforeEach(() => {
+  state = { ...INITIAL_STATE }
+})
+
+describe('when reducing the successful action of setting the wearable preview', () => {
+  const controller = {} as IPreviewController
+
+  it('should return a state with the created wearable preview', () => {
+    expect(
+      previewReducer(state, setWearablePreviewController(controller))
+    ).toEqual({
+      ...INITIAL_STATE,
+      wearablePreviewController: controller
+    })
+  })
+
+  it('should return a state with the unloaded wearable preview', () => {
+    expect(previewReducer(state, setWearablePreviewController(null))).toEqual({
+      ...INITIAL_STATE,
+      wearablePreviewController: null
+    })
+  })
+})
```

Comment:
The `success` keyword is used when there's an action called success, here you can leave it behind.
As the method is executed with a different context, that is, the controller changes, a different describe is needed:
```suggestion
describe('when reducing action of setting the wearable preview', () => {
  const controller = {} as IPreviewController
  it('should return a state with the created wearable preview', () => {
    expect(
      previewReducer(state, setWearablePreviewController(controller))
    ).toEqual({
      ...INITIAL_STATE,
      wearablePreviewController: controller
    })
  })
})
describe('when reducing action of setting the wearable preview to null', () => {
  it('should return a state with the unloaded wearable preview', () => {
    expect(previewReducer(state, setWearablePreviewController(null))).toEqual({
      ...INITIAL_STATE,
      wearablePreviewController: null
    })
  })
})
```
The same applies for the other test.